examples/pennylane_run_quantum_natural_gradient.py

```diff
@@ -0,0 +1,356 @@
+
+.. math:: \theta_{t+1} = \theta_t -\eta \nabla \mathcal{L}(\theta),
+
+where :math:`\mathcal{L}(\theta)` is the loss as a function of
+the parameters :math:`\theta`, and :math:`\eta` is the learning-rate
+or step size. In essence, each optimization step calculates a vector of
+steepest descent direction around the local value of :math:`\theta_t`
+in the parameter space, and updates :math:`\theta_t\rightarrow \theta_{t+1}`
+in by this vector in euclidean space.
+
+The problem with the above approach is that each optimization step
+is highly dependent on the euclidean geometry of the parameter space.
+The parametrization chosen is not necessarily unique; the scale of the
+parameters, or even the parametrization used, could easily be modified.
+If we instead consider the optimization problem as a distribution of possible
+output values given an input, a better approach is to instead perform the
+gradient descent in the *distribution space*, which is invariant with respect
+to the parametrization.
+
+In classical neural networks, the above process is known as
+*natural gradient descent*, and was first introduced by
+`Shun-Ichi Amari (1998) <https://www.mitpressjournals.org/doi/abs/10.1162/089976698300017746>`__.
+The standard gradient descent is modified as follows:
+
+.. math:: \theta_{t+1} = \theta_t - \eta F^{-1}\nabla \mathcal{L}(\theta),
+
+where :math:`F` is the `Fisher information matrix <https://en.wikipedia.org/wiki/Fisher_information#Matrix_form>`__.
+The Fisher information matrix acts as a metric tensor, transforming the
+steepest descent in the euclidean parameter space to the steepest descent in the
+distribution space.
+
+The quantum analogue
+^^^^^^^^^^^^^^^^^^^^
+
+In a similar vein, it has been shown that the standard euclidean geometry
+is sub-optimal for optimization of quantum variational algorithms.
+The space of quantum states instead possesses a unique invariant metric
+tensor known as the Fubini-Study metric tensor :math:`g_{ij}`, which can be used to
+construct a quantum analogue to natural gradient descent:
+
+.. math:: \theta_{t+1} = \theta_t - \eta g^{+}(\theta_t)\nabla \mathcal{L}(\theta),
+
+where :math:`g^{+}` refers to the pseudo-inverse.
+
+.. note::
+
+    It can be shown that the Fubini-Study metric tensor reduces
+    to the Fisher information matrix in the classical limit.
+
+    Furthermore, in the limit where :math:`\eta\rightarrow 0`,
+    the dynamics of the system are equivalent to imaginary-time
+    evolution within the variational subspace, as proposed in
+    `McArdle et al. (2018) <https://arxiv.org/abs/1804.03023>`__.
+"""
+
+##############################################################################
+# Block-diagonal metric tensor
+# ----------------------------
+#
+# The block-diagonal approximation to the Fubini-Study metric tensor
+# of a variational quantum circuit can be evaluated on quantum hardware.
+#
+# Consider a quantum node represented by the variational quantum circuit

```

Comment:
A bit ambiguous here. Both the quantum node and the variational circuit are usually described with a unitary, a state prep, and a measurement setting. Here, you're only describing the unitary

Reply:
Fixed! I've included the state prep, and removed reference to the quantum node (which was out of place)